warden/group/manager_sql.go

```diff
@@ -183,3 +187,33 @@ func (m *SQLManager) ListGroups(limit, offset int) ([]Group, error) {
 
 	return groups, nil
 }
+
+func (m *SQLManager) OverwriteGroupMembers(group string, members []string) error {
+	if err := m.applyInTransaction(m.deleteGroup(group), m.createGroup(group), m.addGroupMembers(group, members)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (m *SQLManager) applyInTransaction(requests ...func(tx *sqlx.Tx) error) error {
```

Comment:
`requests` are really `transactions`, right?

Reply:
A transaction is a sequence of requests I would rather say.

Reply:
How about `executors`?